vallum/src/test/java/VallumTest.java

```diff
@@ -19,5 +24,32 @@ public void test() {
         );
         List<Survey> surveys = vallum.getSurveysByProject("hessenpoort");
         assertEquals(28, surveys.size());
+
+        var firstSurvey = surveys.get(0);
+        Address firstAddress = firstSurvey.getAddresses().get(0);
+        int houseNumber = firstAddress.getHouseNumber();
+        assertEquals(35, houseNumber);
+
+        GridConnection gc = firstAddress.getGridConnections().get(0);
+        Set<PandID> pandIds = gc.getPandIds();
+        assertEquals("[PandID(value=1234567890123456), PandID(value=6543210987654321)]", pandIds.toString());
+
+        var timeSeries = gc.getElectricity().getQuarterHourlyFeedIn_kWh();
+
+        assertTrue(timeSeries.getValues().length > 0);
+
+        for (var survey: surveys) {
+            for (var address: survey.getAddresses()) {
+                for (var gridConnection: address.getGridConnections()) {
+                    var pandIds = gridConnection.getPandIds();
+                    var installedPv_kWp = gridConnection.getSupply().getPvInstalledKwp();
+                    // hydrate model with vars
+                }
+            }
+        }
+
+//        Instant javaStart = timeSeries.getStart().toJavaInstant();
+//        System.out.println(javaStart);
+//        System.out.println(javaStart.isBefore(Instant.parse("2023-01-01T01:00:00+01:00")));
```

Comment:
want to keep the comment or it can be used as a test?

Reply:
This is kinda unfinished, need to refactor ztor to be able to run tests like these.